portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function guessMyTrip()
+{
+  const places=["thailand", "singapore", "tokyo", "australia","peru", "hawaii", "vietnam"];
+  var input =document.getElementById("user_input").value;
+  console.log(input);
+  if(places.includes(input.toLowerCase())){
+      document.getElementById('display').innerHTML = "Well done you guessed it correctly! " + input +" is one of the places I will be travelling to on my Round the World Trip in 2022!";
```

Comment:
column length > 80 which isn't allowed by style guide https://google.github.io/styleguide/jsguide.html#formatting-column-limit. Is there a way to make this multi line?
Example 
var x = "a" +
"b" +
"c"

Reply:
Some ideas http://go/js-style-external#formatting-line-wrapping

Reply:
thanks ,changed it